index.js

```diff
@@ -20,40 +21,42 @@ module.exports = function (tmpl, format) {
 
 		// Do nothing if no contents
 		if (file.isNull()) {
-			this.push(file);
-			return callback();
+			callback(null, file);
```

Comment:
Part of coding style - 1) no sense to do extra checks like `file.isStream()`, `file.isBuffer()` after invoking callback and 2) lower risk of "gotcha" if adding some new code to the function in future.